apps/server/src/modules/authentication/controllers/dto/ldap-authorization.params.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsMongoId, IsString } from 'class-validator';
+
+export class LdapAuthorizationParams {
+	@IsMongoId()
+	@ApiProperty()
+	systemId!: string;
+
+	@IsString()
+	@ApiProperty()
+	username!: string;
```

Comment:
Could be an empty string. I suggest @IsNotEmpty, but it is probably not necessary. If username/password,... is empty and therefor not valid, authentication will fail later anyways.